### PR TITLE
Remove high availability (podmaster) example and note the move to the…

### PR DESCRIPTION
**Pull request by jayunit100**
@mikedanese shall we delete the podmaster example implementation? one more moving part we dont need since there is now the  #16830 implementation which is favored i think.

### CODE REVIEW

**Comment by timothysc** [comment]
+1 to remove.

**Comment by jayunit100** [comment]
@derekwaynecarr can we LGTM this so we can clean it out before it confuses more innocent bystanders :)

**Comment by mikedanese** [comment]
@jayunit100 we just got off holiday today. I will review this soon.

**Comment by jayunit100** [comment]
gotcha thanks! no huge rush ..

File: docs/admin/high-availability.md [hunk 1]
```diff
@@ -277,10 +282,7 @@ set the `--apiserver` flag to your replicated endpoint.
 
 ## Vagrant up!
 
-We indeed have an initial proof of concept tester for this, which is available [here](../../examples/high-availability/).
-
-It implements the major concepts (with a few minor reductions for simplicity), of the podmaster HA implementation alongside a quick smoke test using k8petstore.
-
+Older versions of kubernetes had an examples/high-availabilty vagrant recipe for testing these concepts.  Since these are subsumed by https://github.com/kubernetes/kubernetes/pull/16830, we have now removed this example from current kubernetes source.
```
Comment (mikedanese) [inline docs/admin/high-availability.md]: Can we just remove this section?

File: docs/getting-started-guides/dcos.md [hunk 2]
```diff
@@ -18,11 +18,6 @@
 If you are using a released version of Kubernetes, you should
 refer to the docs that go with that version.
 
-<!-- TAG RELEASE_LINK, added by the munger automatically -->
-<strong>
-The latest release of this document can be found
-[here](http://releases.k8s.io/release-1.1/docs/getting-started-guides/dcos.md).
-
```
Comment (mikedanese) [inline docs/getting-started-guides/dcos.md]: Why did this get touched? It looks like we removed the head of a munge tag without removing the tail.

Comment (jayunit100) [inline docs/getting-started-guides/dcos.md]: Ah weird ok. Lemme redo it

File: docs/admin/high-availability.md (hunk 1 above)
Comment (jayunit100) [inline docs/admin/high-availability.md]: Yeah agree easy enough to remove

**Comment by mikedanese** [comment]
@jayunit100 ping. Can we fix this before we cut v1.2 which might be end of week?

**Comment by jayunit100** [comment]
ah yeah , ill do this now ........

**Comment by jayunit100** [comment]
pushed an update.  should be ok now, hope i did the generated-docs stuff right.. we will see i guess :)

**Comment by mikedanese** [comment]
Awesome. Thanks for fixing up

**Pull request merged**